config/schema_config.yaml

```diff
@@ -73,8 +73,8 @@ epigenomic feature:
     source_url: str
 
 ontology term:
-  is_a: ontology class
   represented_as: node
+  is_a: ontology class
```

Comment:
why is this change here?

Reply:
for most of the nodes the `represented_as` is defined first and have some consistency but I think it is better those changes be in the update prev adapters pr or other pr.